app/models/manageiq/providers/embedded_terraform/automation_manager/configuration_script_source.rb

```diff
@@ -64,7 +65,7 @@ def self.template_name_from_git_repo_url(git_repo_url, branch_name, relative_pat
       basename = File.basename(relative_path)
       parent_path = File.dirname(relative_path)
     end
-    "#{basename}(#{branch_name}):#{hostname}#{path}/#{parent_path}"
+    "#{parent_path}/#{basename}"
```

Comment:
You can also delete line 55 with the hostname.

Reply:
Fixed and repushed